### DISCUSSION TITLE
Welcome to game-copywriter-plans Discussions!

### DISCUSSION THREAD

**Discussion by emperor-tianqi** [emperor-tianqi/game-copywriter-plans#1] [Announcements]
I would like to share my thoughts on the game mode, and welcome any interested friends to participate in the discussion

**Comment by OrionZiK** [comment]
I prefer this game so much. But would you like to design a new module about '_NaNiShiBuShi_' logic?

**Comment by OrionZiK** [comment]
How about adding time backwards travel in this game？
